rules/falco_rules.yaml

```diff
@@ -3077,4 +3077,16 @@
 # Application rules have moved to application_rules.yaml. Please look
 # there if you want to enable them by adding to
 # falco_rules.local.yaml.
+- list: docker_binaries
+  items: [dockerd, containerd-shim, "runc:[1:CHILD]"]
+
+- macro: docker_procs
+  condition: proc.name in (docker_binaries)
 
+- rule: Modify Container Entrypoint
+  desc: Detect file write activities on container entrypoint symlink (/proc/self/exe)
+  condition: >
+    open_write and (fd.name=/proc/self/exe or fd.name startswith /proc/self/fd/) and not docker_procs and container
+  output: >
+    %fd.name is open to write by process (%proc.name, %proc.exeline)
+  priority: WARNING
```

Comment:
During my tests I saw the rule generating false positives on _falcosidekick_ and _argo_ which are two components which can word together this Falco. As we know the open_write isn't that accurate but it's the only option since we can use the write syscall. However the rule address a CVE and it's a good security use case.
My suggestion is to add the rule to the falco ruleset but with the `enabled: false` option to disable by default due to the noise that can generate.